main.js

```diff
@@ -0,0 +1,83 @@
+
+// input fields for activity, minutes, and seconds
+// collect that information on submit, with guards against anything other than numbers in the two latter
+
+// start activity button is submit -- update data model with instance of activity class
+// Activity class needs category, description, minutes, seconds, completed, id
+// on click of start activity, hide form and show timer view -- circle on timer should have same color as category
+// will need to create timer view in html
+
+// error handling for button -- don't let user submit if not all info is complete, but only clear
+// fields if submit is performed
+
+var activityTask = document.getElementById('activity-task');
+var activityMinutes = document.getElementById('activity-minutes');
+var activitySeconds = document.getElementById('activity-seconds');
+var startActivityButton = document.querySelector('.start-activity');
+var startStopButton = document.querySelector('.start-stop-button');
+var allCategoryButtons = document.querySelector('.category-buttons');
+var studyButton = document.querySelector('.icon-study');
+var exerciseButton = document.querySelector('.icon-exercise');
+var meditateButton = document.querySelector('.icon-meditate');
+// ======================================
+
+
+// event listener for buttons (change color of font and change image)
+allCategoryButtons.addEventListener('click', function (event) {
+
```

Comment:
Nice work figuring this out guys - tougher problem than expected! This code is a little repetitious, so if we have time to refactor it later down the line that would be great.